pyproject.toml

```diff
@@ -0,0 +1,26 @@
+[tool.poetry]
+name = "psycopmlutils"
+version = "0.1.0"
+description = "A collection of machine-learnooing utilities used across the psycop-projects."
+authors = ["Martin Bernstorff <martinbernstorff@gmail.com>"]
+
+[tool.poetry.dependencies]
+python = ">=3.8,<3.11"
+black = "^22.3.0"
+numpy = "^1.22.4"
+scipy = "^1.8.1"
+pandas = "^1.4.2"
+SQLAlchemy = "^1.4.37"
+pyodbc = { version = "^4.0.32", markers = "platform_machine == 'x86_64'" }
+pre-commit = "^2.19.0"
+pytest = "^7.1.2"
```

Comment:
Is this everything installed when I install the package? I def. don't want that.

Reply:
Ah, yes! There's a dev subsection, will add it to the next commit.